module4/projects/solo_stretch_tech/index.md

```diff
@@ -3,14 +3,137 @@ title: Solo Stretch Tech
+- **[Circuit Breaker Design Pattern](https://makisushi.io/posts/asd):** Implement the Circuit Breaker design pattern to handle potential failures when calling external APIs or services. Students should use a gem like stoplight or breaker to demonstrate how the circuit breaker prevents cascading failures and improves app resilience. 
+- **Observability:** Integrate an observability tool (e.g., New Relic, Skylight, Honeybadger) to monitor and analyze the application’s performance, errors, and resource usage. Students should configure the tool to track response times, database queries, and bottlenecks.
+</section>
+
+## Timeline + Deliverables
+
+At the end of the days listed below, you will need to turn in certain deliverables. Your cohort calendar specifies the day of the stretch project so you can easily align when each set of deliverables are due. If you fall behind, talk to your instructor as soon as possible. We want to mimic what communication would look like on the job if you needed more time to complete your assigned tasks.
+
+<section class="note">
+### All deliverables will be turned in via **[this google sheet](https://docs.google.com/spreadsheets/d/1dkhqQctc_6nylYyQII22zLU0He_P1k_xfaJOFngV7dM/edit?usp=sharing)**.
+</section>
+
+<section class="dropdown">
+### Day 2
+- Project Proposal: Make a copy of [this document](https://docs.google.com/document/d/1xUWFfFzSJ09GlWi-55942H2RgDJ8GZN-dTMhNsgakig/edit?usp=sharing) and fill it out. Add the link to your proposal to the google sheet. An instructor will review your proposal. You should not move forward until your proposal has been approved. 
+- Stretch Techs: List the stretch techs you have chosen for your project. 
+</section>
+
+<section class="dropdown">
+### Day 4
+
+- FE Design: We expect to see a thought-out design for your frontend before you start coding. Use any tool you'd like (Canva, Figma, Excalidraw, etc) to create a solid wireframe for your app. If you struggle with design, keep it simple. Less is more when it comes to FE design! Look at similar apps out there and take note of what you like about them. Use those features as inspiration.
+- BE Planning Docs: We also expect to see a strong schema design before you start coding. Use any tool you'd like for this. We want to see what data you plan to manage, how it will be organized, and what the relationships between the data will be. 
+- Stretch Tech Learning Resources: Please find at least 2 resources per stretch tech (4 total) that you will use for learning. This could be the official docs, a tutorial, a YouTube series, etc. 
+</section>
+
+<section class="dropdown">
+### Day 7
+
+- Repo(s): Please create your repo(s) and link them in the google sheet. You don't need to have any code in there yet.
+- Project Board: You will link your project board, which should include:
```

Comment:
```suggestion
- Repo(s): Please create your repo(s) and link them in the google sheet. You don't need to have any code in there yet.
- CI/CD: Set up continuous integration on both repos. We recommend using Github actions or CircleCI for this
- Project Board: You will link your project board, which should include:
```

Reply:
I also kind of want to add that they should deploy this via Vercel and Render (or other free service of their choosing) since I think both of those are still free. That would make sense to be due with the Day 7 deliverables as well to encourage Deploy Early, Deploy Often